includes/namespace.php

```diff
@@ -46,6 +46,9 @@ function activate( $network_wide ) {
 		);
 	}
 
+	Story_Post_Type::init();
+	flush_rewrite_rules( false ); // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.flush_rewrite_rules_flush_rewrite_rules
```

Comment:
Should we wrap this in an if statement?
```
if ( !defined( '\WPCOM_IS_VIP_ENV' ) ){
    flush_rewrite_rules( false );
}
```

Reply:
Not familiar with WP.com VIP, so no idea what the best practice is here.

Reply:
Added now.